__init__.py

```diff
@@ -193,9 +248,19 @@ def define_user_vars(self):
 
         return [
             zeekpkg.uservar.UserVar(
-                'name', desc='the name of the package, e.g. "FooBar"'),
+                'name', desc='the name of the package, e.g. "FooBar" or "spicy-http"'),
```

Comment:
`spicy-http` will even be shown if no Spicy-related feature is active. I'd suggest to keep the old format here.

Reply:
I'd like to show the convention for naming Spicy packages. Doesn't seem to hurt to always show it.